projects/storefrontlib/src/lib/cart/cart-shared/order-summary/order-summary.component.scss

```diff
@@ -1,38 +1,32 @@
 @import 'theme';
-@import 'functions';
-@import 'mixins';
 
-// cx-order-summary color variables
-$cx-order-summary-background-color: 'background' !default;
-$cx-order-summary-promotions-color: 'success' !default;
-$cx-order-summary-total-savings-color: 'success' !default;
-
-.cx-order-summary {
-  padding: 1rem;
-  margin-bottom: 1.5rem;
+:host {
+  display: block;
+  padding: var(--cx-padding, 1rem);
+  margin: var(--cx-margin, 0 0 1.5rem 0);
+}
 
-  &__label {
-    text-align: left;
-    padding: 0;
-  }
-  &__amount {
-    text-align: right;
-    padding: 0;
-  }
-  &__promotions {
-    @include var-color('color', $cx-order-summary-promotions-color);
-    @include type('7');
-    padding: 0.5rem 0;
-  }
-  &__row {
-    padding: 0.5rem 0;
-    display: flex;
-    flex-wrap: wrap;
-  }
-  &__total-savings {
-    @include var-color('color', $cx-order-summary-total-savings-color);
-  }
-  &__total-final {
-    font-weight: bold;
-  }
+.cx-summary-label {
+  text-align: var(--cx-text-align, left);
+  padding: var(--cx-padding, 0);
+}
+.cx-summary-amount {
+  text-align: var(--cx-text-align, right);
```

Comment:
One thing I was looking at is how our storefront looks with direction rtl. The thing about having this left and right is that it won't change with the direction. Just wondering if this could be `end` and the other one `start`?

Reply:
Actually out of scope.

Reply:
why not, changed... although we have de default `body` style aligning left, we'll have to check the concept further otherwise, we'll still have to manually change `start/end` as we would've done with `left/right`

Reply:
No problem. I thought it was a good idea but I had second thoughts about if it was needed as part of this ticket. That said, I'm happy it's changed 🙂